tensorflow_datasets/image/icoseg.py

```diff
@@ -0,0 +1,76 @@
+
+"""
+CMU-Cornell iCoseg dataset
```

Comment:
Do you think it will be more meaningful to say "CMU-Cornell iCoseg dataset for image segmentation"

Reply:
Makes sense. I'll change it.